app/models/status.rb

```diff
@@ -10,7 +10,7 @@ class Status < ApplicationRecord
 
   belongs_to :application, class_name: 'Doorkeeper::Application'
 
-  belongs_to :account, inverse_of: :statuses, counter_cache: true
+  belongs_to :account, inverse_of: :statuses, counter_cache: true, touch: true
```

Comment:
Yes, but everytime a status changes anywhere it also does an `UPDATE` for the corresponding account.